package.json

```diff
@@ -15,11 +15,13 @@
     "@types/openfin": "^41.0.0",
     "fs-extra": "^8.0.1",
     "hadouken-js-adapter": "^0.42.2",
+    "openfin-service-signal": "^1.0.0",
+    "openfin-service-tooling": "^1.0.11",
     "rimraf": "^2.6.3",
-    "typescript": "3.5.1",
-    "openfin-service-tooling": "^1.0.11"
+    "typescript": "3.5.1"
   },
   "peerDependencies": {
+    "openfin-service-signal": "^1.0.0",
```

Comment:
For my understanding - why do we add this to `peerDependencies`? What would break if we didn't?

Reply:
Although it's probably really minor it doesn't hurt to make it a peerdependency. It's mainly so we don't bundle the same package twice.
[This explains it better.](https://stackoverflow.com/a/34645112)

Reply:
So this makes sure there's a warning/error if for example, a service is using signals 2.0.0, but also service-config which is using signals 1.0.0?